### PR TITLE
Dzollo/usb uart devc 948 release

### PR DESCRIPTION
**Pull request by denniszollo**

### CODE REVIEW

File: package/ports_daemon/ports_daemon/src/serial.c [hunk 1]
```diff
@@ -139,12 +139,27 @@ static int flow_control_notify(void *context)
 
 int serial_init(settings_ctx_t *settings_ctx)
 {
-  realpath(usb0.tty_path, usb0.tty_path);
-  realpath(usb2.tty_path, usb2.tty_path);
+  char new_path[PATH_MAX];
+
+  /* resolve path to USB0 */
+  char* rp = realpath(usb0.tty_path, new_path);
```
Comment (silverjam) [inline package/ports_daemon/ports_daemon/src/serial.c]: We don't need realpath if we aren't using symlinks anymore

**Comment by silverjam** [comment]
This resolves https://swift-nav.atlassian.net/browse/DEVC-821, does it resolve the issue in DEVC-948?

**Comment by denniszollo** [comment]
I decided to leave the uart order the same for now as to be backwards compatible with our windows usb driver and just get in the minimal working change. 

 If we reorder the usb gadget devices on master, that has to coincide with an update to windows driver.  I'll submit a separate PR for that later.

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.6.12-5-g3c63b5cc
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=596528)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.12-5-g3c63b5cc&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.12-5-g3c63b5cc&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.6.12-5-g3c63b5cc
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.6.12-5-g3c63b5cc)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.6.12-5-g3c63b5cc/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.6.12-5-g3c63b5cc

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.6.12-4-g00a4c190
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=596589)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.12-4-g00a4c190&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.12-4-g00a4c190&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.6.12-4-g00a4c190
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.6.12-4-g00a4c190)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.6.12-4-g00a4c190/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.6.12-4-g00a4c190

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.6.12-4-g00a4c190
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=596598)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.12-4-g00a4c190&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.12-4-g00a4c190&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.6.12-4-g00a4c190
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.6.12-4-g00a4c190)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.6.12-4-g00a4c190/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.6.12-4-g00a4c190

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.6.12-4-g00a4c190
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=596656)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.12-4-g00a4c190&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.12-4-g00a4c190&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.6.12-4-g00a4c190
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.6.12-4-g00a4c190)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.6.12-4-g00a4c190/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.6.12-4-g00a4c190

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.6.12-4-g00a4c190
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=596660)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.12-4-g00a4c190&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.6.12-4-g00a4c190&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.6.12-4-g00a4c190
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.6.12-4-g00a4c190)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.6.12-4-g00a4c190/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.6.12-4-g00a4c190

**Pull request merged**